docs/blog/2020-08-10-drand-launches-v1.0.md

```diff
@@ -0,0 +1,108 @@
+
+In 2019, Cloudflare, EPFL, the University of Chile, Kudelski Security, and Protocol Labs came together to start the League of Entropy and run the drand network. Since then, the group has expanded to include partners in six countries, spanning the spectrum from universities to infrastructure companies. 
+
+### New League of Entropy partners
+
+With the LoE Main Network upgrade, a set of new partners joined the league to increase its robustness and quality of service. The current set of members include:
+
+* [C4DT](https://www.c4dt.org)
+* [ChainSafe](https://chainsafe.io/)
+* [cLabs](https://celo.org/about)
+* [Cloudflare](https://www.cloudflare.com/)
+* [Emerald Onion](https://emeraldonion.org/)
+* [EPFL](https://www.epfl.ch/labs/dedis/)
+* [Ethereum Foundation](https://ethereum.foundation/)
+* [IC3](https://www.initc3.org/)
+* [Kudelski Security](https://www.kudelskisecurity.com/)
+* [Protocol Labs](https://protocol.ai/)
+* [PTisp](https://ptisp.pt/)
+* [Tierion](https://tierion.com/)
+* [UCL](https://www.ucl.ac.uk/)
+* [University of Chile](https://www.uchile.cl)
+
+More partners have signed up to join soon, and we'll keep growing the network over the years to come.
+
+### League of Entropy's governance
+
+Our changes start with the drand community. Drand's strength and resilience are ensured by the number and diversity of its nodes. But a key requirement for the overall security of the network is that no single entity retains complete and centralized control over the drand network. This calls for an explicit and decentralized governance model for the drand community of operators. So one of the first changes was to draft a new model of collaborative governance for the League of Entropy – the consortium supporting and sustaining the drand network.
+
+With this new governance model, we've established the rules and requirements that the League of Entropy needs in order to maintain a high level of network security, ensure that the League operates effectively, and establish the right conditions for ongoing improvement to the production quality of drand. We've also laid down the procedures for adding new members to the League of Entropy in an equitable, fair, and decentralized way. All told, we think these changes set up the drand project for continued success as we find new users and implement new features.
+
+## New website & logo
+
+We built this website to be the drand community's primary repository for the growing body of documentation about drand. We also want this to be the new home for the drand community of developers, operators, and end-users. We'll use it to host our blog and share important updates and news about the project.
+
+In addition to a new site, we also created a logo for the drand project, which you can see at the top of this post or [here](LINK TO LOGO ASSETS FOLDER) for a logo kit. 
```

Comment:
Not sure where this `LINK TO LOGO ASSETS FOLDER` thingy is.

Reply:
I sent this to Alan in Slack to house it in the appropriate place: https://docs.google.com/presentation/d/1xQxONjpVU6DCIf7JhcfrFgejfExHe-NR0qNCuB-zaZQ/edit?usp=sharing